packages/template/renderScript.mts

```diff
@@ -0,0 +1,3 @@
+import {renderVideo} from '@motion-canvas/renderer';
+
+renderVideo('./vite.config.ts');
```

Comment:
I don't like that this is a .mts. It's the only one in the entire codebase. We should rename this to .ts and try to solve the underlying problem.

Reply:
fixed this by making template as a commonjs module